### DISCUSSION TITLE
Two Strings In Footer Text

### DISCUSSION THREAD

**Discussion by mrajalko** [QuestPDF/QuestPDF#173] [Q&A] [answered]
Can we have two strings in the footer one left aligned to show datetime of report generation another right aligned to show current page counter.

**Comment by MarcinZiabek** [comment, marked as answer]
Of course, it is possible 😁 The `Footer` slot behaves like any other container. You can put in there any structure you want: text, images, tables, etc. In your case, you just need to use the `Row` element, like so:

```csharp
page.Footer()
    .Row(row => 
    {
        row.RelativeItem().AlignLeft().Text("Left");
        row.RelativeItem().AlignRight().Text("Right");
    });
```

**Reply by mrajalko** [reply]
Thanks It Worked.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
